src/app/home/podcast/podcast.component.scss

```diff
@@ -0,0 +1,65 @@
+.floating-container {
+  position: fixed;
+  bottom: 0.7rem;
+  left: 0.7rem;
+  background: rgba(255, 255, 255, 0.54);
+  border-radius: 25px;
+  backdrop-filter: blur(4px);
+  width: 16.5%;
+  padding: 6px 6px 0 6px;
+  box-shadow: 0px 4px 4px #cdcdcd;
```

Comment:
I tend to prefer the first one as it's a bit more subtle, but I guess that's a matter of taste. @susiehgt you have an opinion on the matter?

Reply:
<img width="310" alt="Screenshot 2024-12-05 at 09 47 18" src="https://github.com/user-attachments/assets/c11c03fe-078d-41a2-8f41-f6c782852c60">
I improved it a tiny bit with an internal shadow so that we can distinsuish the whole player, if that's what was bothering you @jmedinaebi